src/DataStructures/Tensor/EagerMath/Determinant.hpp

```diff
@@ -126,7 +126,8 @@ struct DeterminantImpl<Symmetry<1, 1>, Index, Requires<Index::dim == 4>> {
 };
 }  // namespace detail
 
-/*! \ingroup Functors
+/*!
+ * \ingroup Tensor
```

Comment:
Consistency question... why put this in group Tensor when you suggested `determinant_and_inverse` be in group NumericalAlgorithms? Seems to me that `determinant` and `determinant_and_inverse` belong together.

Reply:
I agree they should be consistent, but in group Tensor

Reply:
OK by me. If @nilsdeppe agrees I will make the corresponding change in #137

Reply:
I agree! I agree!